docs/core/compatibility/deployment/7.0/x86-host-path.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Breaking change: x86 host path on 64-bit Windows"
+description: Learn about the breaking change in deployment where the 32-bit .NET host isn't added to PATH on 64-bit Windows platforms.
+ms.date: 06/22/2022
+---
+# x86 host path on 64-bit Windows
+
+The x86 versions of .NET installers for Windows have been modified to no longer add the x86 host location (*Program Files (x86)\dotnet*) to the `PATH` environment variable on 64-bit Windows systems.
+
+This change affects .NET Core 3.1, .NET 6, .NET 7, and future versions.
+
+## Previous behavior
+
+The x86 host location was added to `PATH`, even on x64/arm64 systems. Depending on which .NET architecture installer was run first, a user's machine could have either the native (x64/arm64) or x86 host listed first in `PATH`.
+
+## New behavior
+
+Going forward, the x86 host location is only added to the `PATH` environment variable on x86 systems.
```

Comment:
It would be good to clarify that this change only affects the `dotnet` host. It doesn't not affect 32-bit/x86 application hosts, like `myapp.exe`. They will continue to find the x86 runtime (assuming it is installed) correctly.